src/components/IngredientDisplay/IngredientDisplay.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const ContainerStyle: React.CSSProperties = {
+  display: "inline-block",
+  border: "1px solid black",
+  padding: "1rem",
+};
+
+export const IngredientDisplay = ({
+  image,
+  name,
+  id,
+}: {
+  image: string;
+  name: string;
+  id: number;
+}) => (
+  <span style={ContainerStyle}>
```

Comment:
Do you have any thoughts on how to work with the content shift? Maybe a  min width and height to account for the unloaded imgs?